src/plugins/presentation_panel/public/panel_component/presentation_panel.tsx

```diff
@@ -22,10 +22,18 @@ export const PresentationPanel = <
   ApiType extends DefaultPresentationPanelApi = DefaultPresentationPanelApi,
   PropsType extends {} = {}
 >(
-  props: PresentationPanelProps<ApiType, PropsType>
+  props: PresentationPanelProps<ApiType, PropsType> & {
+    hidePanelChrome?: boolean;
+  }
 ) => {
-  const { Component, ...passThroughProps } = props;
+  const { Component, hidePanelChrome, ...passThroughProps } = props;
   const { loading, value, error } = useAsync(async () => {
+    if (hidePanelChrome) {
+      return {
+        unwrappedComponent: isPromise(Component) ? await Component : Component,
```

Comment:
Nice clean way to do this! We can early return and avoid loading all of the extra code if we only want to render the component.